frontend/src/shared/ui/Button/Button.stories.ts

```diff
@@ -1,6 +1,9 @@
 import type { Meta, StoryObj } from "@storybook/react-vite";
 import Button from "./Button";
 
+// Определяем литеральный тип для вариантов кнопки
```

Comment:
non-blocking: комментарий не нужен
```suggestion
// Определяем литеральный тип для вариантов кнопки
```